app/javascript/packages/analytics/Makefile

```
@@ -0,0 +1,3 @@
+digital-analytics-program.js:
+	curl https://raw.githubusercontent.com/digital-analytics-program/gov-wide-code/7c14bb3/Universal-Federated-Analytics.js --silent --output digital-analytics-program.js
+	patch -p1 digital-analytics-program.js digital-analytics-program.patch
```

I'd like to do something here where I define a dependency as being the git SHA (environment variable?), so that we can bust this if we upgrade in the future and someone has a local copy already. Trying to sift through Makefile resources to figure out a nice way to do that, but welcome any advice if someone knows off the top of their head.

---

e.g. something like this (untested, doubtful that it works):

```suggestion
DAP_SCRIPT_SHA = 7c14bb304a18bb1edea6119c27caaf38ffc38064

digital-analytics-program.js: $(DAP_SCRIPT_SHA)
	curl https://raw.githubusercontent.com/digital-analytics-program/gov-wide-code/$DAP_SCRIPT_SHA/Universal-Federated-Analytics.js --silent --output digital-analytics-program.js
	patch -p1 digital-analytics-program.js digital-analytics-program.patch
```

---

this will work
```diff
diff --git a/app/javascript/packages/analytics/Makefile b/app/javascript/packages/analytics/Makefile
index b4eb6ceec5..06b0fea856 100644
--- a/app/javascript/packages/analytics/Makefile
+++ b/app/javascript/packages/analytics/Makefile
@@ -1,3 +1,5 @@
+DAP_SHA ?= 7c14bb3
+
 digital-analytics-program.js:
-	curl https://raw.githubusercontent.com/digital-analytics-program/gov-wide-code/7c14bb3/Universal-Federated-Analytics.js --silent --output digital-analytics-program.js
+	curl https://raw.githubusercontent.com/digital-analytics-program/gov-wide-code/$(DAP_SHA)/Universal-Federated-Analytics.js --silent --output digital-analytics-program.js
 	patch -p1 digital-analytics-program.js digital-analytics-program.patch
```

```
DAP_SHA=abcdef make digital-analytics-program.js
```

---

I think that wouldn't bust the cache and still `make: `digital-analytics-program.js' is up to date`, yeah? 

I came up with c710824fe3, which I _think_ should work? Also seems a little cleaner to separate the download & patching operations.